src/main/kotlin/dnd11th/blooming/api/service/MyPlantService.kt

```diff
@@ -20,29 +20,23 @@ class MyPlantService(
     private val myPlantRepository: MyPlantRepository,
 ) {
     @Transactional
-    fun savePlant(request: MyPlantSaveRequest): MyPlantSaveResponse {
-        if (request.lastWateredDate.isAfter(LocalDate.now()) || request.startDate.isAfter(LocalDate.now())) {
-            throw InvalidDateException(ErrorType.INVALID_DATE)
-        }
+    fun savePlant(
+        request: MyPlantSaveRequest,
+        now: LocalDate,
+    ): MyPlantSaveResponse {
+        verifyDate(request.lastWateredDate, now)
+        verifyDate(request.startDate, now)
 
-        val myPlant =
-            MyPlant(
-                scientificName = request.scientificName,
-                nickname = request.nickname,
-                startDate = request.startDate,
-                lastWateredDate = request.lastWateredDate,
-                alarm =
-                    Alarm(
-                        waterAlarm = request.waterAlarm ?: false,
-                        waterPeriod = request.waterPeriod ?: 3,
-                        nutrientsAlarm = request.nutrientsAlarm ?: false,
-                        nutrientsPeriod = request.nutrientsPeriod ?: 30,
-                        repotAlarm = request.repotAlarm ?: false,
-                        repotPeriod = request.repotPeriod ?: 60,
-                    ),
-            )
+        return MyPlantSaveResponse.from(myPlantRepository.save(request.toMyPlant()))
```

Comment:
```suggestion
        val myPlant = request.toMyPlant()
        return MyPlantSaveResponse.from(myPlantRepository.save(myPlant))
```
축약만이 좋은 건 아닌거 같아요. 한줄에 1. request에서 Plant로 변환, 2. repository에 저장, 3.response 객체로 변환
너무 많은 과정이 있어 가독성이 떨어진다고 생각합니다...!